libcxx/include/experimental/__simd/reference.h

```diff
@@ -71,6 +72,86 @@ class __simd_reference {
 
   template <class _Tp1, class _Storage1, class _Vp1>
   friend void swap(__simd_reference<_Tp1, _Storage1, _Vp1>&& __a, _Vp1& __b) noexcept;
+
+  template <
+      class _Up,
+      enable_if_t<is_assignable_v<value_type&, decltype(std::declval<value_type&>() + std::declval<_Up>())&&>, int> = 0>
```

Comment:
I think this should be reverted to the previous state. I misread what this did before, sorry.